.npmrc

```diff
@@ -1 +1,2 @@
 engine-strict=true
+resolution-mode=highest
```

Comment:
I think this is the default with the latest pnpm

Reply:
This was [copied from the integrations repository](https://github.com/beyonk-group/common/blob/master/apps/integrations/.npmrc).

Reply:
ah ok - can leave for now, it's doing no harm